recipes-soletta/soletta/soletta_git.bb

```diff
@@ -4,7 +4,8 @@
 
 DESCRIPTION = "Soletta library and modules"
 SECTION = "examples"
-DEPENDS = "glib-2.0 libpcre pkgconfig python3-jsonschema-native icu curl"
+DEPENDS = "glib-2.0 libpcre pkgconfig python3-jsonschema-native icu curl libmicrohttpd"
```

Comment:
@barbieri, the patch suggested by @mythi: https://github.com/ostroproject/meta-ostro/pull/62/commits/802e98272680002a5fd39eddc48e5a3b492ceec7 has worked.

We can proceed with this patch without the ">=" in depends that @kad suggested. 

I think it would be better putting this comparison anyway, what do you guys think ?